cmake/kokkos_compiler_id.cmake

```diff
@@ -18,29 +19,37 @@ macro(kokkos_internal_have_compiler_nvcc)
 endmacro()
 
 if(Kokkos_ENABLE_CUDA)
+  # find kokkos_launch_compiler
+  find_program(
+    Kokkos_COMPILE_LAUNCHER
+    NAMES kokkos_launch_compiler
+    HINTS ${PROJECT_SOURCE_DIR}
+    PATHS ${PROJECT_SOURCE_DIR}
```

Comment:
What does it mean to have both HINTS and PATHS with the same directory argument?

Reply:
Agreed